src/Gui/Inventor/Draggers/SoLinearDragger.h

```diff
@@ -68,6 +68,14 @@ class SoLinearDragger : public SoDragger
     SoSFDouble translationIncrement; //!< set from outside and used for rounding.
     SoSFInt32 translationIncrementCount; //!< number of steps. used from outside.
     SoSFFloat autoScaleResult; //!< set from parent dragger.
+    SoSFFloat coneBottomRadius;
+    SoSFFloat coneHeight;
+    SoSFFloat cylinderHeight;
+    SoSFFloat cylinderRadius;
```

Comment:
I wonder if we should expose so much. I think that exposing arrow arrow length and thickness is absolutely enough. That way the proportions will always be consistent. Or maybe there is a a good scenario where exposing everything is useful?

Reply:
Iirc arrow heads of different proportions are used to denote different things in engineering. So, I thought that it might be useful for something in the future. Though, I don't have a strong a preference here. I can make the extra ones protected if you want.